protocol/message.go

```diff
@@ -90,6 +92,34 @@ type MonitoringRequest struct {
 	EndEpoch   uint64 `json:"end_epoch"`
 }
 
+// An AuditingRequest is a message with a CONIKS key directory's address
+// as a string that a CONIKS client sends to a CONIKS auditor, or a CONIKS auditor
```

Comment:
Is this supposed to be used for a auditor to request to a directory? If so, the `DirectoryAddr` field seems to be redundant.